client/cypress/integration/visualizations/choropleth_spec.js

```diff
@@ -36,23 +36,19 @@ describe("Choropleth", () => {
       cy.visit(`queries/${id}/source`);
       cy.getByTestId("ExecuteButton").click();
     });
+    cy.getByTestId("NewVisualization").click();
+    cy.getByTestId("VisualizationType").selectAntdOption("VisualizationType.CHOROPLETH");
   });
 
   it("creates visualization", () => {
-    cy.clickThrough(`
-      NewVisualization
-      VisualizationType
-      VisualizationType.CHOROPLETH
-    `);
-
     cy.clickThrough(`
       VisualizationEditor.Tabs.General
       Choropleth.Editor.MapType
-      Choropleth.Editor.MapType.countries
+      Choropleth.Editor.MapType.Countries
```

Comment:
```suggestion
      Choropleth.Editor.MapType.countries
```

Also, should those other `<Select>` be updated to the `selectAntdOption`? (MapType, KeyType and ValueColumn)

Reply:
This is weird. On my local it was `Choropleth.Editor.MapType.Countries` but in CI it's failing just because of that. 

`selectAntdOption` can be used, but bug with cypress 5 appears only in dialogs (when there's a dialog animation + select animation) so I decided not to replace it everywhere else for now.

Reply:
this is insane, but on local I see
![image](https://user-images.githubusercontent.com/459176/95230360-13330680-0802-11eb-8a2f-7583a84d98ba.png)

@gabrieldutra

Reply:
Ok, now this is weird, leave CI passing and I'll investigate. It can be related to #5186.

Can you try to run `npm run watch` to make sure the last updates are the executed ones?